spiffworkflow-frontend/Dockerfile

```
@@ -18,29 +15,36 @@ RUN apt-get update \
-RUN ./bin/build
+RUN apt-get update && apt-get install -y dos2unix
+
+RUN dos2unix /app/bin/build
+RUN chmod +x /app/bin/build
+RUN /app/bin/build
+
+RUN dos2unix /app/bin/boot_server_in_docker
+RUN chmod +x /app/bin/boot_server_in_docker
 
+RUN cp /app/package.json /app/package.json.bak
+ADD package.json /app/package.json
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)